android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java

```diff
@@ -99,8 +104,15 @@ public void setLocalExtraParameters(final AppLovinMAXAdView view, @Nullable fina
     @Override
     public void onDropViewInstance(AppLovinMAXAdView view)
     {
+        mAdViews.values().remove( view );
+
         view.destroy();
 
         super.onDropViewInstance( view );
     }
+
+    public static AppLovinMAXAdView getAppLovinMAXAdView(final String adUnitId)
+    {
+        return mAdViews.get( adUnitId );
```

Comment:
We should return the underlying `MaxAdView` actually, then `AppLovinMAXModule` can use that

Reply:
Updated.   Now, it will be called:

```
            Class<?> clazz = Class.forName( "com.applovin.reactnative.AppLovinMAXAdViewManager" );
            Method getAppLovinMAXAdViewMethod = clazz.getMethod( "getAppLovinMAXAdView", String.class );
            Object adView = getAppLovinMAXAdViewMethod.invoke( null, "YOUR_ADUNIT_ID" );

            clazz = Class.forName( "com.applovin.reactnative.AppLovinMAXModule" );
            Method getInstanceMethod = clazz.getMethod( "getInstance" );
            Object appLovinMAXModule = getInstanceMethod.invoke( null );
            Method setAmazonResultMethod = clazz.getMethod( "setAmazonResult", Object.class, Object.class );
            setAmazonResultMethod.invoke( appLovinMAXModule, result, adView );
```